book.rb

```diff
@@ -7,4 +7,9 @@ def initialize(title, author, rentals)
     @author = author
     @rentals = rentals
   end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Same as here you should send `person` and `date` as parameters and remember The `book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association you can do:
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```